cfn-resources/trigger/cmd/resource/resource.go

```diff
@@ -257,6 +258,9 @@ func newEventTrigger(model *Model) (*realm.EventTriggerRequest, error) {
 		conf.FullDocument = dTrigger.FullDocument
 		conf.FullDocumentBeforeChange = dTrigger.FullDocumentBeforeChange
 		conf.Unordered = dTrigger.Unordered
+		conf.TolerateResumeErrors = dTrigger.TolerateResumeErrors
+		conf.SkipCatchupEvents = dTrigger.SkipCatchupEvents
```

Comment:
These properties were already defined in our CFN schema but not handled properly?